build_deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -exv
+
+BASE_IMG="bayesian-data-model-importer"
+QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
+IMG="${BASE_IMG}:latest"
+
+GIT_HASH=`git rev-parse --short=7 HEAD`
+
+# build the image
+docker build  --no-cache \
+              --force-rm \
+              -t ${IMG}  \
+              -f ./Dockerfile .
```

Comment:
use Dockerfile.app-sre here
```suggestion
              -f ./Dockerfile.app-sre .
```

Reply:
typo. rectified